PermissionsTestApp/AppDelegate.m

```diff
@@ -52,6 +52,15 @@ - (void)presentPermissionsIfNeeded {
     [permissionsVC setCompletionBlock:^{
         [rootVC reloadPermissions];
     }];
+    
+    __weak ISHPermissionsViewController *weakPermissionsVC = permissionsVC;
+    [permissionsVC setErrorBlock:^(ISHPermissionCategory category, NSError *error) {
+        UIAlertController *alert = [UIAlertController alertControllerWithTitle:ISHStringFromPermissionCategory(category)
+                                                                       message:error.localizedDescription
+                                                                preferredStyle:UIAlertControllerStyleAlert];
+        [alert addAction:[UIAlertAction actionWithTitle:@"Too bad" style:UIAlertActionStyleCancel handler:nil]];
+        [weakPermissionsVC.parentViewController ? weakPermissionsVC : rootVC presentViewController:alert animated:nil completion:nil];
```

Comment:
this is quite tricky to read. `UIViewController *presenter = weakPermissionsVC.parentViewController ? weakPermissionsVC : rootVC;` and then `[presenter present...]` ?